Example/Networks/MainnetAlchemyNetwork.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Web3Swift.io open source project
+// Copyright 2019 The Web3Swift Authors
+// Licensed under Apache License v2.0
+//
+// MainnetAlchemyNetwork.swift
+//
+// Created by Vadim Koleoshkin on 27/03/2019
+//
+
+import Foundation
+import Web3Swift
+
+//Ethereum mainnet via alchemyapi.io service and a Web3Swift.io api key
+public final class MainnetAlchemyNetwork: Network {
+    
+    private let origin: Network = AlchemyNetwork(chain: "mainnet", apiKey: "ETi2ntZoWxd6nTI1qE13Q4I1eLB8AMDl")
```

Comment:
Could we add a comment here that reads:
```swift
// This is a demo key provided by Alchemy for testing purposes only.
// Sign up for a key at https://alchemyapi.io if you plan to do use this in any development, staging, or production environments
```

Reply:
@dtran320 thank you for providing an API key. We added a note about the registration.